diff --git a/node_modules/polyfill-library-3.111.0/lib/index.js b/node_modules/polyfill-library-3.111.0/lib/index.js
index 53902c7..91e20d5 100644
--- a/node_modules/polyfill-library-3.111.0/lib/index.js
+++ b/node_modules/polyfill-library-3.111.0/lib/index.js
@@ -11,6 +11,17 @@ const streamToString = require("stream-to-string");
 const normalizeUserAgent = UA.normalize;
 const LRUCache = require("mnemonist/lru-cache");
 const cache = new LRUCache(1000);
+function compareStrings(a, b) {
+	const n = Math.min(a.length, b.length);
+	for (let i = 0; i < n; i++) {
+		let cmp = a.charCodeAt(i) - b.charCodeAt(i);
+		if (cmp) {
+			return cmp;
+		}
+	}
+
+	return a.length - b.length;
+}
 
 module.exports = {
 	listAliases,
@@ -286,8 +297,8 @@ function getPolyfillString(options_) {
 		).then(() => {
 			// Sort the features alphabetically, so ones with no dependencies
 			// turn up in the same order
-			featureNodes.sort((a, b) => a.localeCompare(b));
-			featureEdges.sort(([, a], [, b]) => a.localeCompare(b));
+			featureNodes.sort(compareStrings);
+			featureEdges.sort(([, a], [, b]) => compareStrings(a, b));
 
 			const sortedFeatures = toposort(featureNodes, featureEdges);
 
